### ISSUE TITLE
docs: cspell in vscode

### ISSUE THREAD

**Issue by mmikhasenko** [ComPWA/compwa-org#140]
Code Checker in VS code does not find `.cspell.json` by default.

It seems to be looking for the `cscpell.json` (no dot) according to
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

likely, the solution is to add the right name to the vs code config, but I have not figured out what

**Comment by redeboer** [comment]
Thanks for reporting.
I also had some problems with this recently. I have the impression it only happens in notebooks, when viewing them with the Jupyter editor. If you open them with the text editor, the problem does not arise.

**Comment by redeboer** [comment]
Yes, this indeed seems to be a problem specific to WSL. The VSCode Jupyter editor does recognise the cSpell config on Ubuntu.

Renaming the config from `.cspell.json` to `cspell.json` also doesn't seem to fix the problem. I have the impression that the VSCode Jupyter editor does not recognise it 'lives' in the workspace folder and searches somewhere else for the config.
